src/activities/binary_bulb/BinaryBulb.qml

```diff
@@ -161,6 +181,16 @@ ActivityBase {
             onHomeClicked: activity.home()
         }
 
+        Score {
+            id: score
+            anchors.bottom: bar.top
+            anchors.right: bar.right
+            anchors.left: parent.left
+            anchors.bottomMargin: 10 * ApplicationInfo.ratio
+            anchors.leftMargin: 10 * ApplicationInfo.ratio
+            anchors.rightMargin: 0
```

Comment:
If rightMargin is 0, I don't think it's needed to add here.

Reply:
it is, as on Score.qml, we put a margin of 30 by default

Reply:
Oops, I didn't remember that :p